site/src/components/Settings/BadgeSettings.vue

```diff
@@ -0,0 +1,189 @@
+                <div class="overflow-hidden h-2 text-xs flex rounded bg-blue-200">
+                    <div
+                        style="width: 30%"
+                        class="
+        shadow-none
+        flex flex-col
+        text-center
+        whitespace-nowrap
+        text-white
+        justify-center
+        bg-blue-500
+      "
+                    />
+                </div>
+                <div class="">
+                    Points PAVE
+                </div>
+            </div>
+        </div>
+        <div class="mt-2">
+            <div class="flex justify-between">
+                <i class="ri-award-line ri-xl" />
+                <i class="ri-medal-line ri-xl" />
+            </div>
+            <div class="relative mt-2">
+                <div class="overflow-hidden h-2 text-xs flex rounded bg-blue-200">
+                    <div
+                        style="width: 60%"
+                        class="
+        shadow-none
+        flex flex-col
+        text-center
+        whitespace-nowrap
+        text-white
+        justify-center
+        bg-blue-500
+      "
+                    />
+                </div>
+                <div class="">
+                    Questions
+                </div>
+            </div>
+        </div>
+        <div class="mt-2">
+            <div class="flex justify-between">
+                <i class="ri-award-line ri-xl" />
+                <i class="ri-medal-line ri-xl" />
+            </div>
+            <div class="relative mt-2">
+                <div class="overflow-hidden h-2 text-xs flex rounded bg-blue-200">
+                    <div
+                        style="width: 20%"
+                        class="
+        shadow-none
+        flex flex-col
+        text-center
+        whitespace-nowrap
+        text-white
+        justify-center
+        bg-blue-500
+      "
+                    />
+                </div>
+                <div class="">
+                    Evenement
+                </div>
+            </div>
+        </div>
+        <div class="mt-2">
+            <div class="flex justify-between">
+                <i class="ri-award-line ri-xl" />
+                <i class="ri-medal-line ri-xl" />
+            </div>
+            <div class="relative mt-2">
+                <div class="overflow-hidden h-2 text-xs flex rounded bg-blue-200">
+                    <div
+                        style="width: 80%"
+                        class="
+        shadow-none
+        flex flex-col
+        text-center
+        whitespace-nowrap
+        text-white
+        justify-center
+        bg-blue-500
+      "
+                    />
+                </div>
+                <div class="">
+                    Nombre d'assos
+                </div>
+            </div>
+        </div>
+        <div class="flex mt-4">
+            <div class="w-1/2 mr-2 border rounded">
+                <div class="text-center m-1">
+                    Affiché sur votre profile
```

Comment:
```suggestion
                    Affiché sur votre profil
```
avec le "e", c'est en anglais ;)